src/Microsoft.Azure.ServiceBus/Message.cs

```diff
@@ -211,9 +200,13 @@ public TimeSpan TimeToLive
 
         // TODO: Calculate the size of the properties and body
         /// <summary>
-        /// Gets the total size of the message in bytes.
+        /// Gets the total size of the message body in bytes.
```

Comment:
Just rewording what I meant - 
Don't want to change the interface/public properties later once we hit GA, and that's why keeping this property public instead of making it internal. 
Now given that we're keeping it public, just making it return BodySize for now instead of 0 till #109 is resolved. Once we work on #109, the same property should return the entire size of message and will also update the summary to mention the same.